docs/topics/api/accounts.rst

```diff
@@ -122,6 +122,27 @@ Other :ref:`editable values <account-edit-request>` can be set at the same time.
     :reqheader Content-Type: multipart/form-data
 
 
+------
+Delete
+------
+
+.. _`account-delete`:
+
+.. note::
+    This API requires :doc:`authentication <auth>` and `Users:Edit`
+    permission to delete accounts other than your own.
+
+.. note::
+    Accounts of users who are authors of Add-ons can't be deleted.
+    All Add-ons (and Themes) must be deleted or transfered to other users first.
+
+This endpoint allows the account to be deleted.  The reviews and ratings
```

Comment:
HTML is anti- double spaces, but I don't know if there are any other general style rules on it.  I can't unlearn it any case, ha.